package.json

```diff
@@ -20,13 +20,16 @@
     "plugin_helpers": "node ../../scripts/plugin_helpers",
     "postbuild": "echo Renaming build artifact to [$npm_package_config_zip_name-$npm_package_version.zip] && mv build/$npm_package_config_id*.zip build/$npm_package_config_zip_name-$npm_package_version.zip"
   },
-  "dependencies": {},
+  "dependencies": {
+    "tough-cookie": "^4.1.3",
+    "web-streams-polyfill": "^4.0.0"
```

Comment:
why do we need these for this change?

Reply:
I tested locally and we can undo these changes

Reply:
addressed